controllers/tutorial.controller.js

```diff
@@ -8,7 +10,7 @@ const getTutorials = async (req, res) => {
     });
     res.status(200).send(tutorials);
   } catch (error) {
-    res.status(500).send({ message: "Internal Server Error" });
+    res.status(500).send({ message: "Tutorial not found" });
```

Comment:
The error should be internal server error because we don't know what's the error.